signature/envelope.go

```diff
@@ -20,22 +20,25 @@ type Envelope interface {
 	SignerInfo() (*SignerInfo, error)
 }
 
-// NewEnvelopeFunc defines a function to create a new Envelope
+// NewEnvelopeFunc defines a function to create a new Envelope.
 type NewEnvelopeFunc func() Envelope
 
-// ParseEnvelopeFunc defines a function to create a new Envelope with given
-// envelope bytes
+// ParseEnvelopeFunc defines a function to create a new Envelope by given
```

Comment:
Not a native speaker, just wondering which is better, `create a new Envelope by give envelope bytes` or ``create a new Envelope from give envelope bytes`` :rofl:

Reply:
good question. I looked through some repos, `ParseEnvelopeFunc defines a function that takes bytes to create an Envelope` might make more sense.